src/applications/personalization/dashboard/actions/messaging.js

```diff
@@ -43,15 +74,6 @@ export const fetchFolder = (id, query = {}) => async dispatch => {
     const folderResponse = await apiRequest(`${baseUrl}${folderUrl}`);
     const messagesResponse = await apiRequest(`${baseUrl}${messagesUrl}`);
 
-    // Escape early if there are errors in either API request.
-    if (folderResponse?.errors || messagesResponse?.errors) {
```

Comment:
This error handling was always in the wrong place. If an error (meaning a non-200 response) occurred with either of the `apiRequest()` calls above, the `apiRequest` will throw a new error that will be handled by the `catch` block below.